<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_artist">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>artist</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_lrc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>lrc</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>256</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeslider">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l_volume">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>volume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_front">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_playpause">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>|&gt;/||</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mute">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_volume">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_nowtime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>nowtime</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_totaltime">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>totaltime</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
